packages/react-instantsearch/src/core/createConnector.test.js

```diff
@@ -134,35 +140,40 @@ describe('createConnector', () => {
         getId,
       })(() => null);
       const unsubscribe = jest.fn();
-      const wrapper = mount(<Connected />, {context: {
-        ais: {
-          store: {
-            getState: () => ({}),
-            subscribe: () => unsubscribe,
+      const wrapper = mount(<Connected />, {
+        context: {
+          ais: {
+            store: {
+              getState: () => ({}),
+              subscribe: () => unsubscribe,
+            },
           },
         },
-      }});
+      });
       expect(unsubscribe.mock.calls.length).toBe(0);
       wrapper.unmount();
       expect(unsubscribe.mock.calls.length).toBe(1);
     });
 
     it('doesn\'t update the component when passed props don\'t change', () => {
-      const getProvidedProps = jest.fn(() => {});
+      const getProvidedProps = jest.fn(() => {
```

Comment:
No reason, I probably add something at some point, removed it, and then the formatter will not join the line back :)